### PR TITLE
Separate two map views

### PR DESCRIPTION
**Pull request by Kylin-Mao**
Kaining: Separeted the original map view controller to two different view controllers.

### CODE REVIEW

File: ZotCovidCare/ZotCovidCare/UCIMapController.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//
```
Comment (felicityzhao99) [inline ZotCovidCare/ZotCovidCare/UCIMapController.swift:1]: Don't forget to write your UCI and Irvine code separately in two files. Good job!

File: ZotCovidCare/ZotCovidCare/IrvineMapController.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import MapKit
+
+class IrvineMapController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
+
```
Comment (Oliviaixinyi) [inline ZotCovidCare/ZotCovidCare/IrvineMapController.swift:18]: darkModeInitialization()

File: ZotCovidCare/ZotCovidCare/UCIMapController.swift [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+//  Created by 茅铠宁 on 2021/3/4.
+//
+
+import UIKit
+
+class UCIMapController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
```
Comment (Oliviaixinyi) [inline ZotCovidCare/ZotCovidCare/UCIMapController.swift:14]: darkModeInitialization()

**Review by Oliviaixinyi — COMMENTED** [review]
Good Job!

**Review by felicityzhao99 — COMMENTED** [review]
Nice work, will merge soon!

**Pull request merged**